server/bin/state/test-26.1.config/pbench-server.cfg

```diff
@@ -0,0 +1,55 @@
+install-dir = %(unittest-dir)s/opt/pbench-server
+
+###########################################################################
+## Deployment section
+###########################################################################
+[pbench-server]
+commit_id = unit-test
+environment = unit-test
+admin-email = unit-test-user@example.com
+pbench-top-dir = %(unittest-dir)s/pbench
+pbench-local-dir = %(unittest-dir)s/pbench-local
+pbench-backup-dir = %(pbench-local-dir)s/archive.backup
+# Add role for sync'ing with satellites
+roles = pbench-prep, pbench-results, pbench-backup, pbench-sync-satellites
+debug_unittest = True
+logger_type = file
```

Comment:
If `logger_type` is indeed supposed to be defined in the `logging` section, then this (and similar for the other two config files) will need to be moved to such a section.